includes/class-wc-payment-tokens.php

```diff
@@ -74,6 +74,7 @@ public static function get_customer_tokens( $customer_id, $gateway_id = '' ) {
 			array(
 				'user_id'    => $customer_id,
 				'gateway_id' => $gateway_id,
+				'limit'      => apply_filters( 'woocommerce_get_customer_payment_tokens_limit', 10 ),
```

Comment:
I have some concerns that if somebody already has `posts_per_page` set to some value greater than 10, that this PR would cause fewer tokens to be returned, and therefore wouldn't be backward-compatible.

What do you think about something like this...?

```suggestion
				'limit'      => apply_filters( 'woocommerce_get_customer_payment_tokens_limit', get_option( 'posts_per_page' ) ),
```